<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spell Guesser 0.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icono.png</normaloff>icono.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="cajaContador">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaImagen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="botonActualizarImagen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Empezar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cajaCampeones">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botonEnviar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menunose">
    <property name="title">
     <string>nose</string>
    </property>
    <addaction name="actionasdf"/>
   </widget>
   <addaction name="menunose"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionasdf">
   <property name="text">
    <string>asdf</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
